core/src/backend/navigator.rs

```diff
@@ -42,14 +42,14 @@ pub enum SocketMode {
 /// The handling mode of links opening a new website.
 #[cfg_attr(feature = "clap", derive(clap::ValueEnum))]
 #[derive(Clone, Copy, Debug, PartialEq, Serialize, Deserialize)]
-pub enum OpenURLMode {
+pub enum OpenLinks {
```

Comment:
I would prefer to just change the translation files.

I think that it makes sense to keep the frontend settings consistent and therefore approve changing the translations. But I think that the underlying variables in the backend should stay as they currently are. Especially the enum should remain `OpenURLMode` since the enum is about the mode of how URLs can be opened. "Open Links" is good as a text displayed to a user in the settings, but it's not a good descriptive name for the underlying variables in the backend.

Reply:
Yeah, all the ftl files, essentially like `open-url-mode = Open Links`.
Although the part of changing the `Confirm` enum value to `Ask` also makes sense and is okay imo, my main problem is that OpenURLMode has been renamed everywhere.